include/exiv2/bmpimage.hpp

```diff
@@ -84,20 +84,11 @@ namespace Exiv2 {
               yet(?) implemented. Calling it will throw an Error(kerWritingImageFormatUnsupported).
          */
         void writeMetadata() override;
-        /*!
-          @brief Todo: Not supported yet(?). Calling this function will throw
-              an instance of Error(kerInvalidSettingForImage).
```

Comment:
Seems like this function remains not implemented, so I don't think this comment was a bad idea, but don't have strong feeling either way.

Reply:
I do not know what I was thinking when I deleted these comments ... I took a look to the BMP format and it seems like EXIF, IPTC are not really supported by BMP. But anyways, it makes sense to indicate what the methods can throw. I will recover these comments :+1: